common/test/acceptance/pages/studio/home.py

```diff
@@ -0,0 +1,21 @@
+"""
+Home page for Studio when logged in.
+"""
+from bok_choy.page_object import PageObject
+
+from common.test.acceptance.pages.studio import BASE_URL
+
+
+class HomePage(PageObject):
+    """
+    Home page for Studio when logged in.
+    """
+
+    url = BASE_URL + "/home"
+
+    def is_browser_on_page(self):
+        self.wait_for_element_presence(
```

Comment:
you shouldn't need a `wait_for_element_presence` under `is_browser_on_page`. `is_browser_on_page` is usually wrapped inside other wait conditions, like `visit` or `wait_for_page`.

So in this case, you'll just want to do `self.q(<foo>).present` or, better, use `.visible` if it is an element that will be visible.